yesod-auth-oauth2.cabal

```diff
@@ -36,6 +36,7 @@ library
                    , aeson                   >= 0.6       && < 0.9
                    , yesod-core              >= 1.2       && < 1.5
                    , authenticate            >= 1.3.2.7   && < 1.4
+                   , random
```

Comment:
No real reason. I'm not too familiar with UUIDs and any benefits they may provide over a random token. I'm fine with removing the `uuid` dependency.